core/wiki/config/DefaultMissingType.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/config/DefaultMissingType
+
```

Comment:
If the code above uses `.getTiddlerText(title,defaultText)` we do not need to create this tiddler. The config tiddler only needs to be defined if it is needed.

Reply:
Deleted tiddler from core.